admin_guide/web_console_customization.adoc

```diff
@@ -224,3 +224,25 @@ oauthConfig:
 
 Relative paths are resolved relative to the master configuration file. You must
 restart the server after changing this configuration.
+
+== Changing the Logout URL
+
+You can change the location console users will be sent to upon logging out of
+the console by modifying the `*logoutURL*` parameter in the
+*_/etc/openshift/master/master-config.yaml_* file:
+
+====
+----
+...
+assetConfig:
+  logoutURL: "www.example.com"
+...
+----
+====
+
+This can be useful when authenticating with
+link:configuring_authentication.html#RequestHeaderIdentityProvider[Request
```

Comment:
Give the full path per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#internal-cross-references